### PR TITLE
Fix mac build

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Fix mac build -- Not sure what this `?` was about, but it was causing problems because the `?` gets uri encoded or something?
https://app.circleci.com/pipelines/github/AudiusProject/audius-client/8074/workflows/ccd03ada-c871-42bc-aba0-0ef7ff508888/jobs/68533

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

It's a good guess

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/eslint-config-audius/package.json [hunk 1]
```diff
@@ -21,26 +21,26 @@
     "test": "echo \"Error: run tests from root\" && exit 1"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^5.30.6",
```
Comment (dylanjeffers) [inline packages/eslint-config-audius/package.json:24]: wait seriously this fixes stuff?

**Review by dylanjeffers — APPROVED** [review]

File: packages/eslint-config-audius/package.json (hunk 1 above)
Comment (raymondjacobson) [reply packages/eslint-config-audius/package.json:24]: nono sorry, it's this

```
    "eslint-plugin-jest": "?26.5.3",
```

we should pin versions anyway though

**Review by dylanjeffers — CHANGES_REQUESTED** [review]
wait actually isn't this related to lock-file issues with npm version?

File: packages/eslint-config-audius/package.json (hunk 1 above)
Comment (raymondjacobson) [reply packages/eslint-config-audius/package.json:24]: the question mark is exploding things

Comment (dylanjeffers) [reply packages/eslint-config-audius/package.json:24]: oh nice nice, it looks like we should probably also fix the lock-file while we're at it?

Comment (raymondjacobson) [reply packages/eslint-config-audius/package.json:24]: wait what's wrong with the lock-file?

**Review by dylanjeffers — APPROVED** [review]
looks like we figured out why, and lockfile stuff will be resolved when we migrate ourselves to v16

File: packages/eslint-config-audius/package.json (hunk 1 above)
Comment (raymondjacobson) [reply packages/eslint-config-audius/package.json:24]: understanding this now is b/c xcode 14 has a node 16 dep -- will wait till we just get on node 16 errwher

**Pull request merged**